eclair-core/src/main/scala/fr/acinq/eclair/tor/Socks5Connection.scala

```diff
@@ -236,4 +247,10 @@ object Socks5ProxyParams {
     } else {
       proxyParams.credentials_opt
     }
+
+  def proxyAcceptsConnections(proxyParams: Socks5ProxyParams): Boolean = Try {
```

Comment:
This is only used in tests, can you move it to the `BlockchainWatchdogSpec` file?